doc/authors.md

```diff
@@ -0,0 +1,44 @@
+# Authors
+
+This is a list of people that contributed to PermaplanT,
+including their responsibilities and focus.
+
+## Markus Raab
+
+- email: markus@permaplant.net
+- devel/test on: Debian stable
+- responsibilities: lead
+
+## Yvonne Markl
+
+- email: yvonne@permaplant.net
+- responsibilities: biology
+
+## Samuel
+
+- focus group: UX, Polyculture
+- responsibilities: Nextcloud
+
+## Gabriel
+
+- responsibilities: Backend
+
+## Moritz
+
+- focus group: DB, Geometry/GIS, Polyculture
+- responsibilities: -||-
+
+## Giancarlo
+
+- focus group: UX
+- responsibilities: Frontend
+
+## Nursultan
+
+- focus group: DB
+- frontend tests, react
```

Comment:
responsibilities field is missing here

Reply:
Thank you, I added responsibilities everywhere.